frontend/app/src/components/helper/display/icons/GeneratedIcon.vue

```diff
@@ -50,12 +50,18 @@ const textStyle = computed<Style>(() => {
   const length = get(text).length;
   const { value } = get(dimensions);
 
-  const fontSize = (value - 2) / Math.max(2.5, length);
+  const fontSize = (value - 2) / Math.max(length, length < 2 ? 1.8 : 2.5);
```

Comment:
1.8 and 2.5 seem very specific values. How do you calcula them?

Reply:
no explanation, I just pick numbers that looks good for the size (many trials and errors)